apps/climatemappedafrica/src/components/Navigation/ExploreNavigation/index.js

```diff
@@ -72,8 +72,9 @@ function ExploreNavigation({ explorePagePath, logo, variant }) {
             <DropdownSearch
               icon={SearchIcon}
               href={explorePagePath}
-              placeholder="Search for a Location" // TODO: Read from cms
+              placeholder="Search for a Location"
```

Comment:
Shouldn't this come from the CMS?

Reply:
Since we do not expect any other label update for this soon, we can leave it as is.